### PR TITLE
Push devfile json schema to docs

### PR DESCRIPTION
**Pull request by davidfestal**
### What does this PR do?

This PR provides automatic pushes of updated devfile Json schema to its place in the docs attachments.

It is triggered each time a push to master is done

### What issues does this PR fix or reference?

This PR fixes issue https://github.com/devfile/api/issues/174

### Is your PR tested?

Yes

### CODE REVIEW

**Review by JPinkney — APPROVED** [review]
LGTM

**Comment by mmulholla** [comment]
Looks like the update commits the change directly to the docs master branch, would this trigger and regeneration of  devfile.github.io or would that need a separate pull and merge request?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -56,4 +56,51 @@ jobs:
...
+          lastCommit="$(cd ../api-repo; git log -1 --format=%H)"
+          lastCommitterName="$(cd ../api-repo; git log -1 --format=%an)"
+          lastCommitterEmail="$(cd ../api-repo; git log -1 --format=%ae)"
+          
+          git config --global user.email "${lastCommitterEmail}"
+          git config --global user.name "${lastCommitterName}"
+          
+          git add docs/modules/ROOT/attachments/jsonschemas/next/devfile.json
+          git commit -m "Update devfile schema based on devfile/api@${lastCommit}"
+          git push "https://devfile-ci-robot:${{ secrets.DOCS_UPDATE_SECRET}}@github.com/devfile/docs"
```
Comment (amisevsk) [inline .github/workflows/ci.yaml:105]: Is the inconsistent spacing (`${{ secrets.DOCS_UPDATE_SECRET}}`) intentional?

**Review by amisevsk — APPROVED** [review]
LGTM

**Review by amisevsk — APPROVED** [review]
You re-requested review <1m after I submitted :)

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (davidfestal) [reply .github/workflows/ci.yaml:105]: Not it isn't

Comment (davidfestal) [reply .github/workflows/ci.yaml]: Fixed here: https://github.com/devfile/api/pull/176/commits/89ac24be8d917ced183675d0bb9fed59af6b3bf0

**Pull request merged**